config/kernel/linux-rockchip64-edge.config

```diff
@@ -390,7 +387,7 @@ CONFIG_ARM64_ERRATUM_2054223=y
 CONFIG_ARM64_ERRATUM_2067961=y
 CONFIG_ARM64_ERRATUM_2441009=y
 CONFIG_ARM64_ERRATUM_2457168=y
-CONFIG_ARM64_ERRATUM_2645198=y
+# CONFIG_ARM64_ERRATUM_2645198 is not set
```

Comment:
Same comment as on the current version.  I would think this erratum should be included, but I didn't dig into what specifically it addresses.

Reply:
Here is the list: https://github.com/torvalds/linux/blob/master/Documentation/arch/arm64/silicon-errata.rst

That one which I switched to `not set` is for Armv9 Cortex A715 cores.
There already are several errata for A510/A710 cores already flagged, but since we have no boards yet with SoCs with such cores I believe they can be counterproductive: at worst they may hamper performances trying to circumvent some broken hardware features on such amrv9 cores which are not broken on armv8.

This is the list of the enabled ones on the current kernel:

![image](https://github.com/armbian/build/assets/15271078/51f0361b-72c4-4589-9535-ac23139cfc6d)

Perhaps a benchmark disabling the unneeded errata would be nice to gather some numbers and see if the impact is significant or negligible (I bet it is negligible), yet many out-of-order A7x cores get the errata of in-order A5x cores and viceversa. Albeit absolutely acceptable since we can't have a kernel for each board and some SoCs are big.LITTLE, is a bit "smelly" :nose:

Reply:
I was just worried that this was a merge error and an oversight.  If you believe this is the right selection, I'm fine with your reasoning.

Reply:
four eyes are better than two! :wink: